install.sh

```diff
@@ -115,6 +139,9 @@ tty_red="$(tty_mkbold 31)"
 tty_bold="$(tty_mkbold 39)"
 tty_reset="$(tty_escape 0)"
 
+
+unset HAVE_SUDO_ACCESS # unset this from the environment
+
```

Comment:
This line will drop any environment hacking for the previous installer. Such as:

```bash
NONINTERACTIVE=1 HAVE_SUDO_ACCESS=1 /bin/bash -c "$(
	curl -fsSL https://github.com/Homebrew/install/raw/HEAD/install.sh |
	sed -E 's#^(\s*)(HOMEBREW_PREFIX_DEFAULT)=(.*)$#\1\2="${HOME}/.linuxbrew"#'
)"
```

Note that, in non-interactive mode, we don't support installing Homebrew to any prefix other than `/home/linxbrew/.linuxbrew` on Linux with this installer. Users, who don't have `sudo` privileges, should use the [Untar-anywhere](https://docs.brew.sh/Installation#untar-anywhere) method to install Homebrew non-interactively on Linux.